discord/user.py

```diff
@@ -123,6 +125,16 @@ def default_avatar(self):
         """:class:`Asset`: Returns the default avatar for a given user. This is calculated by the user's discriminator."""
         return Asset._from_default_avatar(self._state, int(self.discriminator) % len(DefaultAvatar))
 
+    @property
+    def banner(self):
```

Comment:
```suggestion
    def banner(self) -> Optional[Asset]:
```

Reply:
As the rest of the properties (i.e `avatar`) had no return annotations, I opted out of typing up these ones.

Reply:
That's just because user isn't typed yet.
Feel free to annotate